app/posts/apply-for-teacher-training/2020-06-18-international-candidates.md

```diff
@@ -0,0 +1,244 @@
+
+* understand the teacher training system – the [Aspiring Teacher Facebook Forum](https://www.facebook.com/groups/1357146377672255/) and [Get Into Teaching](https://getintoteaching.education.gov.uk/) website are not optimised to appear in search results outside of the UK
+
+* give information about residency status and qualification equivalencies
+
+* spend money on things like equivalency statements, document translation and travel to interviews
+
+* wait around for translations and equivalence certificates
+
+Some of this friction is inevitable. Other parts are being addressed by our colleagues in Get Into Teaching, Find and international policy.
+
+There are some aspects we can address through the application process.
+
+## Residency and visa status
+
+If a candidate does not already have the right to work or study in the UK there are 2 visa routes: a general work visa ([Tier 2](https://www.gov.uk/tier-2-general)) for salaried courses, or a general student visa ([Tier 4](https://www.gov.uk/tier-4-general-visa)).
+
+Our [initial design for residency status](/apply-for-teacher-training/residency-status/) assumed that candidates would know which visa they require.
+
+However, research with international candidates suggested that it’s too early in the process to ask this question. Candidates have not necessarily considered these details yet, and financial circumstances (and therefore visa requirements) might change between application and acceptance.
+
+It may be useful for international applicants to know which providers can sponsor a visa, but this information should be available earlier in the journey – possibly on Find.
+
+### User need
+
+{% from "user-needs/macro.njk" import appUserNeeds %}
+{{ appUserNeeds({
+  items: collections['user-need'] | slugs(['AN025'])
+}) }}
+
+### Hypothesis
+
+If we allow international candidates to defer entering details about visas
+Then they can continue with their application
+We’ll know this works when the proportion of international candidates who submit a complete application compared to those who start, is comparable to the same value for domestic candidates.
+
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+{{ appScreenshots({
+  headingLevel: 3,
+  items: [{
+    text: "Personal details",
+    caption: "Ask the nationality question in a separate step and remove the question about languages spoken."
+  }, {
+    text: "What is your nationality?",
+    caption: "* British and Irish – no further questions are asked\n* Other – candidate asked to select nationality from structured list\n* I have multiple nationalities – candidate can give details as free text"
+  }, {
+    text: "Do you have the right to work or study in the UK? (Yes)",
+    caption: "Candidates who are not British or Irish are asked to give details about their residency status, of they know it. They can give details of their residency, or indicate that this is something they will need to consider later."
```

Comment:
about their residency status, **if** they know it

Reply:
Good catch, fixed!